src/core/PathwaysFilterBar.tsx

```diff
@@ -25,21 +25,17 @@ import { useCoreStore } from "./CoreStoreProvider";
 import DetailsGroup from "./DetailsGroup";
 import DownloadDataButton from "./DownloadDataButton";
 import MethodologyLink from "./MethodologyLink";
-import { PopulationFilters } from "./types/filters";
+import { EnabledFilters, PopulationFilters } from "./types/filters";
 import { getFilterOption } from "./utils/filterOptions";
-import { CORE_PATHS, CORE_VIEWS } from "./views";
+import { CORE_PATHS } from "./views";
 
 const PopulationFilterBar: React.FC<{
-  view: keyof typeof CORE_VIEWS;
   filterOptions: PopulationFilters;
-}> = ({ filterOptions, view }) => {
-  const { filtersStore, pageProjectionsStore } = useCoreStore();
+  enabledFilters?: EnabledFilters;
+  handleDownload: () => Promise<void>;
```

Comment:
Nit: put optional fields after required fields
```suggestion
  handleDownload: () => Promise<void>;
  enabledFilters?: EnabledFilters;
```

Reply:
Good call! Thanks.